oshi-core/src/main/java/oshi/software/os/OSFileStore.java

```diff
@@ -349,4 +349,22 @@ public boolean updateAtrributes() {
         }
         return false;
     }
+
+    /** {@inheritDoc} */
+    @Override
+    public String toString() {
+        StringBuilder sb = new StringBuilder();
```

Comment:
You're not including any punctuation between the entries here.   Also I'd prefer to keep the formatted output (GB rather than the entire integer) and the percentage from the [existing output](https://github.com/oshi/oshi/blob/fe457f3e4e5b1e4bc37b4401aa600ddea042da9c/oshi-core/src/test/java/oshi/SystemInfoTest.java#L296) here.